src/data/polynomial/eval.lean

```diff
@@ -154,6 +154,20 @@ begin
   simp [mul_comm]
 end
 
+lemma eval₂_mul_eq_zero_of_left (q : polynomial R) (hp : p.eval₂ f x = 0) :
+  (p * q).eval₂ f x = 0 :=
+begin
+  rw eval₂_mul f x,
+  exact mul_eq_zero_of_left hp (q.eval₂ f x)
+end
+
+lemma eval₂_mul_eq_zero_of_right (hq : q.eval₂ f x = 0) (p : polynomial R) :
```

Comment:
I would keep the arguments in the same order as for the lemma above it. So first `p` and then `hq`.
```suggestion
lemma eval₂_mul_eq_zero_of_right (p : polynomial R) (hq : q.eval₂ f x = 0) :
```